usr/share/rear/lib/layout-functions.sh

```diff
@@ -318,10 +318,8 @@ find_disk() {
 }
 
 find_disk_and_multipath() {
-    res=$(find_disk "$1")
-    if [[ -n "$res" || "$AUTOEXCLUDE_MULTIPATH" =~ ^[yY1] ]]; then
-        echo $res
-    else
+    find_disk "$1"
+    if ! [[ "$AUTOEXCLUDE_MULTIPATH" =~ ^[yY1] ]]; then
```

Comment:
Optionally when you are at it:

Things like `[[ "$VAR" =~ ^[yY1] ]]` is old code
See https://github.com/rear/rear/wiki/Coding-Style
for our current coding style recommendations.


Since a longer time we have the `is_true` and `is_false` functions
see usr/share/rear/lib/global-functions.sh
https://raw.githubusercontent.com/rear/rear/master/usr/share/rear/lib/global-functions.sh
how to use them.

Reply:
Thanks, fixed in the updated patch.